libs/server-sdk/include/launchdarkly/server_side/config/built/data_system/lazy_load_config.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <launchdarkly/server_side/data_interfaces/sources/iserialized_pull_source.hpp>
+
+#include <chrono>
+#include <memory>
+
+namespace launchdarkly::server_side::config::built {
+
+struct LazyLoadConfig {
+    /**
+     * \brief Specifies the action taken when a data item's TTL expires.
+     */
+    enum class EvictionPolicy {
+        /* No action taken; eviction is disabled. Stale items will be used
+         * in evaluations if they cannot be refreshed. */
+        Disabled = 0
+    };
+
+    EvictionPolicy eviction_policy;
+    std::chrono::milliseconds eviction_ttl;
+    std::shared_ptr<server_side::data_interfaces::ISerializedDataPullSource>
```

Comment:
This is the thing anyone could implement (`ISerializedDataPullSource`). 

The `shared_ptr` is pretty general - we can have a higher level constructor to make a Redis client object, or let people pass in their own.

We could also have `source` be a `variant<shared_ptr<ISerializedDataPullSource>, SomeRedisArgs>` instead if we wanted to defer construction to SDK creation time and totally encapsulate it. 

But, I think the current way is nice because then the Client itself has zero knowledge that it's dealing with Redis.